### PR TITLE
coverage: Add missing files/dirs to phpunit configs

### PR DESCRIPTION
**Pull request by anomiex**
## Proposed changes:

Ideally everything would be under `src/`, but that's not the case for a bunch of our packages and plugins.

Also annoying is that PHPUnit doesn't allow globs in `<file>`, so we have to list any root-level files individually.

There are still some root-level files not touched here that might be, e.g. `actions.php` files in a few packages or most plugins' entry point files. Not sure if those should be included or not.
### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

None

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Check the coverage data, see if these are covered now? Kind of hard to do though.

### CODE REVIEW

File: projects/plugins/boost/phpunit.xml.dist [hunk 1]
```diff
@@ -17,6 +17,10 @@
 
 	<filter>
 		<whitelist processUncoveredFilesFromWhitelist="false">
```
Comment (tbradsha) [inline projects/plugins/boost/phpunit.xml.dist:19]: As best I can tell, the above is [the same as the default](https://github.com/sebastianbergmann/php-code-coverage/blob/34cc484af1ca149188d0d9e91412191e398e0b67/src/CodeCoverage.php#L56-L64), right? Or am I missing some nuance as to why we need this attribute?

```suggestion
		<whitelist>
```

Comment (anomiex) [reply projects/plugins/boost/phpunit.xml.dist:19]: Yeah, all these could use updating. For now I just copied this from `tools/cli/skeletons/`

File: projects/plugins/jetpack/phpunit.xml.dist [hunk 2]
```diff
@@ -115,14 +115,53 @@
...
+			<file>locales.php</file>
+			<file>uninstall.php</file>
 			<directory suffix=".php">3rd-party</directory>
 			<directory suffix=".php">extensions</directory>
 			<directory suffix=".php">_inc</directory>
 			<directory suffix=".php">json-endpoints</directory>
 			<directory suffix=".php">modules</directory>
 			<directory suffix=".php">sal</directory>
 			<directory suffix=".php">src</directory>
+			<directory suffix=".php">views</directory>
```
Comment (tbradsha) [inline projects/plugins/jetpack/phpunit.xml.dist:164]: Skipping `tests` probably makes sense, but perhaps we should include `tools`?

Comment (anomiex) [reply projects/plugins/jetpack/phpunit.xml.dist:164]: I doubt we'll ever write tests for the tools stuff. They're build scripts that happen to be written in PHP, not part of the plugin itself.
https://github.com/Automattic/jetpack/blob/1b5acbd9977992e5f2bf3c57c772e8013837a20c/projects/plugins/jetpack/.gitattributes#L67

**Review by tbradsha — APPROVED** [review]

**Pull request merged**